src/main/java/com/globalitgeeks/examninja/usermanagement/dto/ErrorResponse.java

```diff
@@ -2,11 +2,17 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
+import java.util.Map;
 @Data
+@NoArgsConstructor
+@AllArgsConstructor
 public class ErrorResponse {
-    private String error;
-    public ErrorResponse(String error){
-        this.error=error;
+    private String message;
+    private Map<String, String> error;
+
+    public ErrorResponse(String message) {
```

Comment:
You need to remove this constructor now.

Reply:
Addressed 
Removing the single arg constructor may break the code for two custom business exceptions- IncorrectPassword for login and UserNotFoud for login and change password api,.